### DISCUSSION TITLE
Newgrounds Logo at the beginning

### DISCUSSION THREAD

**Discussion by Cheemsandfriends** [FunkinCrew/Funkin#1689] [Ideas]
Hello lol.
So I was thinking a little bit.
How about instead just an image of the Newgrounds Logo Make like a small animation like the Gf Cheer?

**Comment by ninjamuffin99** [comment]
nope. cuz this is a newgrounds game

**Reply by Cheemsandfriends** [reply]
> 
> 
> nope. cuz this is a newgrounds game

Yeah But I meant like instead of an image, A small animation of Tankman Cheering
